packages/base/src/utils.ts

```diff
@@ -242,3 +242,13 @@ export function remove_buffers(
   const new_state = remove(state, []) as JSONObject;
   return { state: new_state, buffers: buffers, buffer_paths: buffer_paths };
 }
+
+export const SVG_ICON = `<svg style="height:50%;max-height: 50px;" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48">
```

Comment:
It's unclear from the variable name `SVG_ICON` what is this icon. Maybe rename it to `FILE_ERROR_SVG_ICON` or `BROKEN_FILE_SVG_ICON` or just `ERROR_SVG_ICON`?